### PR TITLE
- Copying License.txt from source packages (#11)

### PR DESCRIPTION
**Pull request by moozzyk**
- Updating LibuvCopier to 1.1
- Fixing NuGet.config

### CODE REVIEW

File: tools/LibuvCopier/project.json [hunk 1]
```diff
@@ -4,19 +4,16 @@
...
-    "Newtonsoft.Json": "8.0.3"
+    "Newtonsoft.Json": "9.0.1"
   },
   "frameworks": {
-    "netcoreapp1.0": {
+    "netcoreapp1.1": {
       "dependencies": {
         "Microsoft.NETCore.App": {
-          "version": "1.0.0-rc2-3002702",
+          "version": "1.1.0-*",
```
Comment (pranavkm) [inline tools/LibuvCopier/project.json]: Could we pin these to the LTS builds (1.0.0)? That way we can remove the reference to the aspnetcore feed from NuGet.config.

File: src/Libuv/project.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     ],
     "files": {
       "mappings": {
-        "/": "../../content/License.txt",
+        "/": "License.txt",
```
Comment (pranavkm) [inline src/Libuv/project.json:18]: Does the version need to change here?

**Review by pranavkm — APPROVED** [review]

File: tools/LibuvCopier/project.json (hunk 1 above)
Comment (moozzyk) [reply tools/LibuvCopier/project.json]: Sure.

File: src/Libuv/project.json (hunk 2 above)
Comment (moozzyk) [reply src/Libuv/project.json:18]: I assume you are referring to Microsoft.NETCore.Platforms": "1.1.0",. It does not have to change - should it stay at 1.0.1?

Comment (pranavkm) [reply src/Libuv/project.json:18]: Sorry, I meant the version for this package. It's currently at `1.9.0`

Comment (moozzyk) [reply src/Libuv/project.json:18]: Oh that. It's going to be a separate change. 1.10.0 build packages will have a slightly different structure and more changes is required.

**Comment by moozzyk** [comment]
:up::date:

File: NuGet.config [hunk 3]
```diff
@@ -1,8 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
-    <add key="CoreFx" value="https://dotnet.myget.org/f/dotnet-core/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
```
Comment (pranavkm) [inline NuGet.config:4]: Does it build if you remove this now? Maybe it'd be nice that Libuv doesn't depend on AspNet's artifacts at all.

**Review by pranavkm — APPROVED** [review]

**Review by muratg — APPROVED** [review]

File: NuGet.config (hunk 3 above)
Comment (moozzyk) [reply NuGet.config:4]: In this repo we put the final package together by taking the OS specific packages from the feed so, no you can't get rid of this file. I did get rid of NuGet.config file in the libuv-build repo where they were not needed.

**Pull request merged**